frontends/web/src/components/copy/Copy.tsx

```diff
@@ -62,9 +62,8 @@ export const CopyableInput = ({ alignLeft, alignRight, borderLess, value, classN
   };
 
   const copy = () => {
-    textAreaRef.current?.focus();
-    textAreaRef.current?.select();
```

Comment:
these got lost in the rebase, but maybe they were only required because of  the old method of copying, and not anymore with the new clipboard.writeText? @thisconnect

Reply:
correct, not needed anymore. old way dependent on an form input element that has focus, new way just passes a string